### PR TITLE
Fix monitor definition

### PR DESCRIPTION
**Pull request by yaodingyd**
### What does this PR do?

Fix monitor definition. These monitor definition are validated using /api/v1/monitor/validate API

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: nomad/assets/monitors/nomad_heartbeats_received.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	"options": {
 		"notify_audit": false,
 		"locked": false,
-		"timeout_h": 60,
+		"timeout_h": 1,
```
Comment (yaodingyd) [inline nomad/assets/monitors/nomad_heartbeats_received.json:12]: `timeout_h` should be 1-24. I think here 60 means 60m, so change to 1(h)

File: nomad/assets/monitors/nomad_job_is_failing.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
 	"options": {
 		"notify_audit": false,
 		"locked": false,
-		"timeout_h": 60,
+		"timeout_h": 1,
```
Comment (yaodingyd) [inline nomad/assets/monitors/nomad_job_is_failing.json:12]: same as above

File: php_apcu/assets/monitors/php-apcu_expunges.json [hunk 3]
```diff
@@ -20,10 +20,6 @@
         "thresholds": {
             "critical": 1,
             "critical_recovery": 0
-        },
-        "threshold_windows": {
```
Comment (yaodingyd) [inline php_apcu/assets/monitors/php-apcu_expunges.json:24]: `threshold_windows` only works for anomaly monitor(this one is metric)

File: php_apcu/assets/monitors/php-apcu_high_usage.json [hunk 4]
```diff
@@ -20,10 +20,6 @@
         "thresholds": {
             "critical": 90,
             "warning": 80
-        },
-        "threshold_windows": {
```
Comment (yaodingyd) [inline php_apcu/assets/monitors/php-apcu_high_usage.json:24]: `threshold_windows` only works for anomaly monitor(this one is metric)

File: php_opcache/assets/monitors/php-opcache_expunges.json [hunk 5]
```diff
@@ -20,10 +20,6 @@
         "thresholds": {
             "critical": 1,
             "critical_recovery": 0
-        },
```
Comment (yaodingyd) [inline php_opcache/assets/monitors/php-opcache_expunges.json:23]: `threshold_windows` only works for anomaly monitor(this one is metric)

**Review by brett0000FF — APPROVED** [review]

**Review by nmuesch — COMMENTED** [review]
Looks good to me once the:
>     excessiveblockedHTTP.json must have an `integration` tag
issue is addressed.

**Comment by yaodingyd** [comment]
@irabinovitch @withgod @signalsciences Could you take a look? This PR is to fix existing monitors with broken definition. Each one is commented with its specific error

**Comment by withgod** [comment]
@yaodingyd 👍
looks fine. thanks.

**Comment by anawishnoff** [comment]
Hi @signalsciences, heads up that we'll be merging this PR in on Monday. Let us know if you have any concerns. These changes are simply to make the monitors pass validations, and will not cause changes for customers.

**Comment by nmuesch** [comment]
Based on the last message from Ana, I'm going ahead and merging this. Please let us know if you experience any issues.

**Pull request merged**